.github/workflows/yum.yml

```diff
@@ -77,6 +77,17 @@ jobs:
           --env CENTOS_STREAM=${{ matrix.centos-stream }} \
           ${{ matrix.test-docker-image }} \
           /fluentd/fluent-package/yum/serverspec-test.sh
+      # Confluent community is not supported on AmazonLinux.
```

Comment:
> Confluent community is not supported on AmazonLinux.

Sorry, could you please tell me what the comment means?

Reply:
It is a garbage. It should be removed.